index.html

```diff
@@ -0,0 +1,263 @@
+                          </ul>
+                        </div>
+                        <p>
+                        A smart assistant to make driving more safe, 
+                        efficient, and fun by unlocking 
+                        your most expensive computer: your car.</p>
+
+                        <div class="tags">
+                          <ul>
+                            <li>html</li>
+                            <li>Ruby on Rails</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                          </ul>
+                        </div>
+
+                        <a href="#" class="btn-one">See project</a>
+                      </li>
+                    </ul>
+                  </div>
+               </li>
+          </ul>
+  </section>
+
+<section class="section-about">
+<div class="container">
+<div class="about-grid">
+    <div class="text">
+      <div class="headline-text">
+          <h1 class="second-text">About Myself</h1>
+
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a
+             project you need coded, don’t hestiate to contact me.</p>
+
+            <a href="#" class="btn-connect"><h6>Let's connect</h6></a>
+          <ul class="medium">
+              <li class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-medium"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-github"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></li>
+          </ul>
+          <br>
+          <a href="#" class="btn-one">Get My Resume</a>
+          </div>
+       </div>
+
+    <div class="text">
+    <ul class="stacks">
+        <li class="languages">
+            
+            <a href="#"> <h2>Languages</h2></a>
+            <ul class="grid-container">
+            <li>
+                
+                <img src="image/js.png" alt="js_logo">
+                <p>JavaScript</p>
+                
+            </li>
+            <li>
+                
+                <img src="image/html.png" alt="html_logo">
+                <p>HTML</p>
+                
+            </li>
+                
+            <li class="langi">
+                
+                <img src="image/css.png" alt="css_logo">
+                <p>CSS</p>
+                
+            </li>
+                
+            
+          </ul>
+        </li>
+        
+        <li>
+            <a href="#"> <h2>Frameworks</h2> </a>
+        </li>
+        
+        <li>
+            
+          <a href="#"> <h2>Skills</h2> </a>
+  
+        </li>
+    </ul>
+    </div>
+</div>
+</div>
+</section>
```

Comment:
- [ ] Kindly, follow the figma to fix this as well, You only have to change the padding between your JavaScript, CSS and HTML as well as changing the length of their divs, for more details:

------------------------- Original ----------------------------

![langu](https://user-images.githubusercontent.com/55635977/126050560-701eabfb-5e14-4a78-820d-7cedbf7d3856.png)
------------------------- Yours ----------------------------
![langua](https://user-images.githubusercontent.com/55635977/126050568-d56dd404-1abc-40e3-9529-c32ba6286aab.png)